transport/tchannel/channel_outbound_test.go

```diff
@@ -176,12 +176,25 @@ func TestChannelCallSuccess(t *testing.T) {
 			assert.True(t, ok, "deadline expected")
 			assert.WithinDuration(t, time.Now(), dl, 200*testtime.Millisecond)
 
-			err = writeArgs(call.Response(),
-				[]byte{
-					0x00, 0x01,
-					0x00, 0x03, 'f', 'o', 'o',
-					0x00, 0x03, 'b', 'a', 'r',
-				}, []byte("great success"))
+			if flipCoin() {
```

Comment:
Same here, let's drop this randomness so we can have deterministic tests.

Reply:
Resolved.